### ISSUE TITLE
Paging prompts appear if dashboard is made to show in user's custom gdbinit

### ISSUE THREAD

**Issue by 10110111** [cyrus-and/gdb-dashboard#73]
I have a custom `~/.gdbinit.d/gdbinit`, which contains the following:
```
# Reset prompt to the more familiar one
dashboard -style prompt '(gdb)'
```
On x86-64 this results in paging prompts on first appearance of dashboard.
It might be a good idea to `run('set pagination off')` earlier than sourcing custom `gdbinit`s.

**Comment by cyrus-and** [comment]
I cannot reproduce this one, `dashboard -style ...` commands display the dashboard only if the inferior program is running, i.e., not at startup. What am I missing?

**Comment by 10110111** [comment]
Ah, right, I start GDB like this:
```
gdb /bin/true -ex 'b *0' -ex r -ex 'd 1' -ex 'source ~/packages/gdb-dashboard/.gdbinit'
```

**Comment by cyrus-and** [comment]
Alright, yes, the fact is that the dashboard isn't really meant to be loaded on demand. Moving the `run('set pagination off')` before surely avoids this annoying issue but every `dashboard -style ...` command would trigger a redraw, which is annoying as well.

A possible solution would be to disable the dashboard updates during the parsing of custom init files.

**Comment by cyrus-and** [comment]
Implemented in 44234a33502e691304908147f51e4855478ff0c4, give it a try.

**Comment by 10110111** [comment]
Thanks, seems to work.
